### PR TITLE
Add option for android color hex format

### LINKED ISSUE #79: support more format option for Android color export
When exporting color in Android project, we can only export them in `rgba` format.
would be nice if user can specify them as `argb` format.

### PR DESCRIPTION
**Pull request by lichin-lin**
## Description

As mentioned in #79 , there should provide a way for user to choose either export colors in `rgba` or `arbg` format.

## Changes
* Added params to the yaml file: `hexFormat`
* Change private extension Color's hex (in AndroidColorExporter.swift) from variable to a function.

Fixes #79

### CODE REVIEW

**Review by subdan — CHANGES_REQUESTED** [review]
Hello. Thanks for the PR. I know that colors in android always specified in ARGB format. It is an FigmaExport error to export colors for android in RGBA format. So, we don’t need a new parameter (android.color.hexFormat). Just change RGBA to ARGB format in the `AndroidColorExporter.swift`.

**Comment by lichin-lin** [comment]
Ok 🙂 thanks for the explanation! let me do a update tomorrow and send the PR again.

**Comment by lichin-lin** [comment]
@subdan I've update the code, please review when have time, thanks 😀

File: README.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ Example of these files:
 
 ### Android
 
-Colors will be exported to `values/colors.xml` and `values-night/colors.xml` files.
+Colors will be exported to `values/colors.xml` and `values-night/colors.xml` files in `argb` format.
```
Comment (subdan) [inline README.md:201]: It is a default format. Discard this change.

**Review by subdan — CHANGES_REQUESTED** [review]
Discard changes to the README.md.

**Review by subdan — APPROVED** [review]

**Pull request merged**